net/tcp/Kconfig

```diff
@@ -50,11 +50,11 @@ config NET_TCPURGDATA
 		compiled in. Urgent data (out-of-band data) is a rarely used TCP feature
 		that is very seldom would be required.
 
-config NET_TCP_CONNS
-	int "Number of TCP/IP connections"
-	default 8
+config NET_TCP_CONNS_PER_BLOCK
+	int "Number of TCP/IP connections per block(one for each open)"
+	default 4
 	---help---
-		Maximum number of TCP/IP connections (all tasks)
+		Maximum number of TCP/IP connections per block(all tasks)
```

Comment:
The  number of TCP/IP connections per allocation(all tasks)

Reply:
What aboutn NET_{UDP|TCP}_CONNS_PER_BLOCK -> NET_{UDP|TCP}_CONNS_PER_ALLOC  so it is consistent with this behavior that is not the same as the FD change that is blocks?

Reply:
Done